docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -251,16 +206,13 @@ spec:
         name: multipathd.service
 ----
 +
-If you are using labels to restrict the nodes where Block Storage
-services are running you need to use a `MachineConfigPool` as described in
-the xref:about-node-selector_planning[About node selector] to limit the effects of the
-`MachineConfig` to only the nodes where your services may run.
+If use labels to restrict the nodes where Block Storage services are running, you need to use a `MachineConfigPool` to limit the effects of the `MachineConfig` to only the nodes where your services run. For more information, see xref:about-node-selector_planning[About node selector].
 +
 If you are using a single node deployment to test the process, replace `worker` with `master` in the `MachineConfig`.
 +
 To configure the Block Storage services to use multipathing, enable the
-`use_multipath_for_image_xfer` configuration option in all the backend sections
-and in the `[DEFAULT]` section for the backup service. This is the default in control plane deployments. Multipathing works as long as the service is running on the {OpenShiftShort} host. Do not override this option by setting `use_multipath_for_image_xfer = false`.
+`use_multipath_for_image_xfer` configuration option in all the back-end sections
+and in the `[DEFAULT]` section for the backup service. This is the default in control plane deployments. For multipathing to work, the service must run on the {OpenShiftShort} host. Do not override this option by setting `use_multipath_for_image_xfer = false`.
```

Comment:
The service must run on the RHOSP host, or the RHOSO host?

Reply:
In the context of this Multipathing section we are describing using MachineConfig to prepare the RHOCP hosts, so I think the following sentence is accurate:
"For multipathing to work, the service must run on the RHOCP host."